WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -159,7 +159,7 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     private func copyToSharedDefaultsIfNeeded() {
-        if !AppConfiguration.isJetpack && FeatureFlag.sharedUserDefaults.enabled && !UserPersistentStore.standard.isOneOffMigrationComplete {
+        if !AppConfiguration.isJetpack && FeatureFlag.contentMigration.enabled && !UserPersistentStore.standard.isOneOffMigrationComplete {
```

Comment:
I think the method `copyToSharedDefaultsIfNeeded` is no longer used, so perhaps you can also remove this method along with the planned change for `UserPersistentStoreFactory` above?

Reply:
Yeah definitely. It will have no use for us.